source/2019-07-19-the-ember-times-issue-107.md

```diff
@@ -33,69 +34,90 @@ responsive: true
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [Custom Configuration of Ember Auto Import for Your Project 🐹](https://github.com/ef4/ember-auto-import/tree/v1.3.0#customizing-build-behavior)
```

Comment:
Did you want to use the custom emoji above?

Reply:
```suggestion
## [Custom Configuration of Ember Auto Import for Your Project ⏬](https://github.com/ef4/ember-auto-import/tree/v1.3.0#customizing-build-behavior)
```